src/www/ui/api/documentation/openapi.yaml

```diff
@@ -797,6 +797,79 @@ paths:
                           "fossy"        
         default:
           $ref: '#/components/responses/defaultResponse'
+  /users/tokens:
+    post:
+      operationId: createRestApiToken
```

Comment:
Please deprecate the old endpoint if there is a new endpoint to create tokens.

https://github.com/fossology/fossology/blob/5eaca363ca51c212038939acc98c24672b6392e0/src/www/ui/api/documentation/openapi.yaml#L60

Reply:
I think there are two different use-cases for these endpoints. `/tokens` is called when a user tries to login so we provide the `username` and `password` in the request body. `/token/users` route which I created is used to create a customised REST API token directly from the Admin page without the need of any username or password.

Also, if we want to merge functionalities of these two routes there are only two parameters i.e., `username` and `password` which is required and not required in either cases, I would like to know how to confirm whether the user the user is already authenticated and requesting for the creation of new token and in other case the user is not authenticated and we need to check the `username` and `password`?

Reply:
[UPDATE]: Marked the old endpoint as deprecated. @GMishx